### ISSUE TITLE
Logo in dark mode is gray-on-black

### ISSUE THREAD

**Issue by krugerk** [beeminder/BeeSwift#150]
```[tasklist]
### Desiderata
- [ ] Is the answer just "Beeminder doesn't support dark mode currently"?
```

![signin dark on dark](https://user-images.githubusercontent.com/4656811/90789551-5515f500-e307-11ea-88b5-8f9837fb7ddc.jpg)

Here is a screenshot, which actually is more legible now as a screenshot than the screen was in the app at the time (perhaps brightness was also very low).

## Cognata

* #55 
* #176

_**Verbata**: accessibility, contrast, dark mode, visibility,_

**Comment by krugerk** [comment]
Between the logo and 'I have an account' there is also some black text which cannot be seen here on the black background.

**Comment by emeraldfleur** [comment]
Submitted PR #561 to attempt to address issue. The black text mentioned previously by krugerk has been changed to white in a previous update, so would argue that this PR brings the logo more in line with the current color schema in the app of white on black. Also, your eyes will immediately feel relieved seeing the logo in white, in my opinion.

**Comment by krugerk** [comment]
Feel free to add the text 
Fixes #150
in the description of your PR

See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests
